Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM debian:jessie
```

Comment:
Do you know if docker is available on the Beagle Bone?  It's ARM.  It would be great if it was.  Either way, a non BB docker would be good for developers, but they will ultimately need a BB to run the pin IO.